pldoc/_includes/examples/dropdown-menu-js.html

```diff
@@ -53,7 +53,7 @@
                 url: '/'
             },
             button: {
-                icon: 'icon-angle-down',
+                icon: 'fa-angle-down',
```

Comment:
Do you need to also include ``fa`` because the dropdown doesn't add it by default.